aptos-move/move-examples/rewards_pool/simple/sources/rewards.move

```diff
@@ -0,0 +1,97 @@
+module rewards::rewards {
+    use std::signer;
```

Comment:
nit: Sort the imports by highlighting these lines and choose Edit > Sort in IntelliJ